drivers/SmartThings/matter-appliance/src/matter-refrigerator/init.lua

```diff
@@ -161,6 +166,7 @@ end
 local function refrigerator_tcc_supported_modes_attr_handler(driver, device, ib, response)
   local refrigeratorTccModeSupportedModes = {}
   for _, mode in ipairs(ib.data.elements) do
+    clusters.RefrigeratorAndTemperatureControlledCabinetMode.types.ModeOptionStruct:augment_type(mode)
```

Comment:
I don't yet understand why, but the other files have a version check around similar code